lib/db/mysql/index.js

```diff
@@ -177,7 +200,75 @@ MysqlStore.prototype = {
       return client;
     });
   },
+  registerClientDeveloper: function regClientDeveloper(developerId, clientId) {
+    if (!developerId || !clientId) {
+      var err = new Error('Owner registration requires user and developer id');
+      return P.reject(err);
+    }
+
+    var rowId = unique.id();
+
+    logger.debug('registerClientDeveloper', {
+      rowId: rowId,
+      developerId: developerId,
+      clientId: clientId
+    });
 
+    return this._write(QUERY_CLIENT_DEVELOPER_INSERT, [
+      buf(rowId),
+      buf(developerId),
+      buf(clientId)
+    ]);
+  },
+  getClientDevelopers: function getClientDevelopers (clientId) {
+    if (! clientId) {
+      return P.reject(new Error('Client id is required'));
+    }
+
+    return this._read(QUERY_CLIENT_DEVELOPER_LIST_BY_CLIENT_ID, [
+      buf(clientId)
+    ]);
+  },
+  activateDeveloper: function activateDeveloper(email) {
+    if (! email) {
+      return P.reject(new Error('Email is required'));
+    }
+
+    var developerId = unique.developerId();
+    logger.debug('activateDeveloper', { developerId: developerId });
+    return this._write(QUERY_DEVELOPER_INSERT, [
+      developerId, email
+    ]);
+  },
+  getDeveloper: function(email) {
+    if (! email) {
+      return P.reject(new Error('Email is required'));
+    }
+
+    return this._readOne(QUERY_DEVELOPER, [
+      email
+    ]);
+  },
+  removeDeveloper: function(email) {
+    if (! email) {
+      return P.reject(new Error('Email is required'));
+    }
+
+    return this._write(QUERY_DEVELOPER_DELETE, [
+      email
+    ]);
+  },
+  developerOwnsClient: function devOwnsClient(developerEmail, clientId) {
+    return this._readOne(QUERY_DEVELOPER_OWNS_CLIENT, [
+      developerEmail, buf(clientId)
+    ]).then(function(result) {
+      if (result) {
+        return P.resolve(result);
```

Comment:
As above, though it looks like here, the query will return the `rowId`. Whichever result is returned, it should be consistent, or we'll get errors.